test/factory/common_factory_tests.dart

```diff
@@ -126,7 +126,7 @@ void commonFactoryTests(ReactComponentFactoryProxy factory) {
   });
 }
 
-void domEventHandlerWrappingTests(ReactDomComponentFactoryProxy factory) {
+void domEventHandlerWrappingTests(ReactComponentFactoryProxy factory) {
```

Comment:
?

Reply:
loosened to allow testing with the new ReactJsComponentFactoryProxy